ocpp/charge_point.py

```diff
@@ -74,13 +74,39 @@ def remove_nones(dict_to_scan):
     }
     return dict_to_scan
 
+async def add_latency(delay=0, frequency=0.0):
+    """
+    Adds latency to a given function.
+    We will use Gaussian Distribution to add some variance to the amount of latency passed.
+    We will randomly affect functions to have latency or not by weighing the choice of adding
+    latency or not.
+
+    delay (int): average (?) delay to add to call results MILISECONDS
+    frequency (float): percentage of times the delay will be added [0.0, 1.0]
+    """
+    async def decorator(function):
+        async def wrapper(*args, **kwargs):
+            result = await function(*args, **kwargs)
+            """
+            import numpy as np
```

Comment:
I think that making numpy as a dependency of this is way to heavy

Reply:
Yes, I am aware. In no way was I thinking of adding numpy here. And I'm still thinking if I can find another way to not use it at all.